packages/zosfiles/CHANGELOG.md

```diff
@@ -2,8 +2,11 @@
 
 All notable changes to the Zowe z/OS files SDK package will be documented in this file.
 
-## `8.10.3`
 
+## Recent Changes
+- Enhancement: The `Copy.dataset` function now creates a new data set if the inputted target data set does not exist. [#2349](https://github.com/zowe/zowe-cli/issues/2349)
```

Comment:
Can we use another word besides "inputted"? Like "entered" or "selected" or "included"?